packages/core/src/sync-store/index.ts

```diff
@@ -211,7 +207,7 @@ export const createSyncStore = ({
               await tx
                 .insertInto("blockFilters")
                 .values(fragment)
-                .onConflict((oc) => oc.column("id").doUpdateSet(fragment))
+                .onConflict((oc) => oc.column("id").doNothing())
                 .executeTakeFirstOrThrow();
```

Comment:
I don't think you need the `executeTakeFirstOrThrow` here, could probably cause issues